kiosk_skeleton/boot/kioskbrowser.ini

```diff
@@ -7,6 +7,8 @@ hostname = "kioskpi"
 ;force_resolution = "1920x1080"
 ; force a custom modelines (for specialty diplays like embedded monitors, car screens, etc.)
 ;custom_modeline = "40.141 1024 1032 1064 1104 600 604 612 618 +HSync -VSync"
+; can be used for forcing screen rotation on raspberry pi boards (normal, left, right, inverted)
```

Comment:
This will work on most devices, not only Raspberry Pi's, so that comment could be updated.

Reply:
Updated